src/test/java/OrderSmokeTests.java

```diff
@@ -1,50 +1,59 @@
 import controller.OrderController;
-import io.restassured.response.Response;
-import models.Order;
-import org.testng.Assert;
+import io.qameta.allure.Description;
+import io.qameta.allure.testng.AllureTestNg;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Listeners;
 import org.testng.annotations.Test;
-import static utils.Constants.*;
 
+import static utils.Constants.DEFAULT_ORDER;
+import static utils.Constants.NON_EXIST_ID_INT;
+
+@Listeners({AllureTestNg.class})
 public class OrderSmokeTests extends BaseTest {
 
     OrderController orderController = new OrderController();
 
     @Test
+    @Description("Add order test with default order")
     public void createOrderTest() {
-        Response response = orderController.addDefaultOrder();
-        response.prettyPrint();
-        Assert.assertEquals(response.statusCode(), SUCCESS_STATUS_CODE);
+        orderController.addDefaultOrder()
+                .statusCodeIs(HttpStatus.SC_OK)
+                .jsonValueIsNotNull("shipDate")
```

Comment:
It's better to leave at least 1 test when you check not all fields using jsonValueIs method, but comparing 2 objects. In some cases when it's should compare all fields it's even better and faster. To align with Allure you need to create such method and annotate with @Step. If there will be mismatch - you should return actual and expexted objects.